src/ert/cli/main.py

```diff
@@ -43,16 +44,25 @@ def run_cli(args: Namespace, plugin_manager: Optional[ErtPluginManager] = None)
     # Changing current working directory means we need to update
     # the config file to be the base name of the original config
     args.config = os.path.basename(args.config)
+    forward_models = []
 
     ert_config = ErtConfig.with_plugins().from_file(args.config)
 
     local_storage_set_ert_config(ert_config)
+    all_fm_steps = [fms.name for fms in ert_config.forward_model_steps]
```

Comment:
This should be `Counter(fms.name for fms in ert_config.forward_model_steps)` so that the Counter is only created once.